prerender/app.py

```diff
@@ -135,16 +137,32 @@ def _save_to_cache(key: str, data: bytes, format: str = 'html') -> None:
     if not skip_cache:
         try:
             data = await cache.get(url, format)
+            modified_since = await cache.modified_since(url)
```

Comment:
Might be better to return `Optional[float]`? when it returns `None` don't set these headers.